java/getting-started/licensing/_index.md

```diff
@@ -5,105 +5,114 @@ weight: 90
+### **File**
 
-**Example 1**
+The easiest method of setting a license requires you to place the license file in the folder containing Aspose.Slides.jar or your applications' jar.
 
-In this example, Aspose.Slides will attempt to find the license file in the folder that contains the JARs of your application.
+This Java code shows you how to set a license file:
 
 ``` java
+// Instantiates the License class
 com.aspose.slides.License license = new com.aspose.slides.License();
 
+// Sets the license file path
 license.setLicense("Aspose.Slides.Java.lic");
 ```
 
-**Example 2**
+{{% alert color="warning" %}} 
+
+If you place the license file in a different directory, when you call the [SetLicense](https://reference.aspose.com/slides/java/com.aspose.slides/License#setLicense-java.lang.String-) method, the license file name at the end of the specified explicit must be the same as your license file.
+
+For example, you can change the license file name to *Aspose.Slides.Java.lic.xml*. Then, in your code, you have to pass the path to the file (ending with *Aspose.Slides.Java.lic.xml*) to the [SetLicense](https://reference.aspose.com/slides/java/com.aspose.slides/License#setLicense-java.lang.String-) method.
+
+{{% /alert %}}
+
+### **Stream**
 
-Initializes a license from a stream.
+You can load a license from a stream. This Java code shows you how to apply a license from a stream:
 
 ``` java
+// Instantiates the License class
 com.aspose.slides.License license = new com.aspose.slides.License();
 
+// Sets the license through a stream
 license.setLicense(new java.io.FileInputStream("Aspose.Slides.Java.lic"));
 ```
 
-## **Setting a License in PHP Using PHP/Java Bridge**
-Setting the license in PHP using **PHP/Java Bridge** is similar to **Aspose.Slides for Java**. For this reason, PHP developers actually use  **Aspose.Slides for Java** API in PHP. **PHP/Java Bridge** provides an interface to the PHP developers that makes it possible to use Java classes in PHP syntax.
-
-``` php
-<?php
-	 //Using aspose.slides.jar file so that the classes inside the jar file
-	 //can be used
-	 java_require("aspose.slides.jar");
-
-	 try
-	 {
-	   //Create a stream object containing the license file
-	   $fistream=new Java("java.io.FileInputStream", "Aspose.Slides.Java.lic");
-
-	   //Instantiate the License class
-	   $license=new Java("com.aspose.slides.License");
-
-	   //Set the license through the stream object
-	   $license->setLicense($fistream);
-
-	   //Closing the stream
-	   $fistream->close();
-	 }
-	 catch(JavaException $ex)
-	 {
-	   //Printing the exception, if it occurs
-	   echo $ex->toString();
-	 }
-?>
-```

```

Comment:
@EmSeriki

I think we should avoid using the PHP Java Bridge code directly in our documentation, because we have Aspose.Slides for PHP via Java.

Reply:
Alright, thank you.